security_monkey/auditors/route53.py

```diff
@@ -0,0 +1,50 @@
+#     Licensed under the Apache License, Version 2.0 (the "License");
+#     you may not use this file except in compliance with the License.
+#     You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+"""
+.. module: security_monkey.auditors.route53
+    :platform: Unix
+
+.. version:: $$VERSION$$
+.. moduleauthor::  Alex Cline <alex.cline@gmail.com> @alex.cline
+
+"""
+from security_monkey.auditor import Auditor
+from security_monkey.watchers.route53 import Route53
+from security_monkey import app
+import re
+
+
+class Route53Auditor(Auditor):
+    index = Route53.index
+    i_am_singular = Route53.i_am_singular
+    i_am_plural = Route53.i_am_plural
+    internal_record_regex = [
+        r"^10\.",
+        r"^172\.16\.",
```

Comment:
RFC-1918 declares this range to be a /12, which means this regex is incorrect.

It's everything from 172.16.0.0 up to 172.31.255.255.

This should probably be done with the IPNetwork class:

https://github.com/Netflix/security_monkey/blob/b81b15b838b3576fb4b66b81b37a69f51acf51e1/security_monkey/auditors/security_group.py#L45

```
def _check_rfc_1918(cidr):
        """
        EC2-Classic SG's should never use RFC-1918 CIDRs
        """
        if ipaddr.IPNetwork(cidr) in ipaddr.IPNetwork('10.0.0.0/8'):
            return True

        if ipaddr.IPNetwork(cidr) in ipaddr.IPNetwork('172.16.0.0/12'):
            return True

        if ipaddr.IPNetwork(cidr) in ipaddr.IPNetwork('192.168.0.0/16'):
            return True

        return False
```